.travis.yml

```diff
@@ -28,6 +28,8 @@ install:
 
 # Build steps
 script:
+  - sed -i 's/nestingLimit = 64/nestingLimit = 1024/g' third_party/capnproto/c++/src/capnp/message.h
```

Comment:
I think modifying the header in the third_party/ is a bad idea, as we'd typically get the header from the system.

Instead, whenever we use the `RenderOptions`, we should conffigure it accordingly.

Reply:
UHDM and Surelog both have their own pull from capnproto (git recursive update).
I do the sed after the recursive update. 
For a manual install I updated the instructions to do the same. The capnp proto on most Ubuntu install is obsolete, that is why I have it as a git pull.

Reply:
I see now how that can be configured in the reader.
I'll try your suggestion.